sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryClient.java

```diff
@@ -110,7 +110,7 @@ public String getSchemaId(String schemaGroup, String schemaName, String schemaSt
      * @return The unique identifier for this schema.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<String> getSchemaIdWithResponse(String schemaGroup, String schemaName, String schemaString,
+    Response<String> getSchemaIdWithResponse(String schemaGroup, String schemaName, String schemaString,
```

Comment:
The async client has `getSchemaIdWithResponse` as a public API. We should make this public too or hide this method in async client.